### ISSUE TITLE
dark dialogue is hard to see

### ISSUE THREAD

**Issue by weepy** [TheComputerM/svelte-materialify#175]
there dialogue in dark-mode is hard to see (i assume because it usually uses shadows to show up). 
Not sure what a good solution is here => perhaps a slight light border ?

**Comment by Florian-Schoenherr** [comment]
Good idea, I think we should follow the material specs here and if still needed provide a solution for styling this better.
You could try to make a custom solution with some css for now.

**Comment by weepy** [comment]
yeah the specs are "These surface overlay values are designed to maximize
legibility, while ensuring the different elevation levels are discernible
from one another."

[image: image.png]

css is fine - though is it possible to apply without the "Advanced Install"
- (i had problems with getting this to work) ?

**Comment by weepy** [comment]
sorry. - i meant that the "elevation" uses a lighter color in dark mode
rather than the shadow.

**Comment by Florian-Schoenherr** [comment]
well, with global css it should be possible. The minimal install is probably more for trying out and REPL.
